resolvers/projectResolver.ts

```diff
@@ -1027,12 +1027,37 @@ export class ProjectResolver {
       .setParameter('slugs', slugs)
       .getMany()
 
+
     const projectsUpdatedListing = projects.map(project => {
       return { id: project.id, listed: listed }
     })
 
+    const segmentProject = await projects.map( async project => {
```

Comment:
You don't use `segmentProject` so you can just call `await projects.map...` without  putting the result in the `segmentProject`, simply you can remove `const segmentProject = ` from this line